relays/client-kusama/src/codegen_runtime.rs

```diff
@@ -0,0 +1,8338 @@
+// Copyright 2019-2023 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Autogenerated runtime API
+//! THIS FILE WAS AUTOGENERATED USING parity-bridges-common::runtime-codegen
+//! EXECUTED COMMAND: target/debug/runtime-codegen --from-node-url wss://kusama-rpc.polkadot.io:443
+
+#[allow(dead_code, unused_imports, non_camel_case_types)]
+#[allow(clippy::all)]
+pub mod api {
```

Comment:
if we were to move `relays` to `polkadot-sdk` repo, we'd no longer need these huge generated files, right?

Reply:
and we'd have better control on the runtime API version used too

Reply:
Yes, we'll be able to use the actual runtimes directly from the repo.